databaseQueries/organizationExtension.js

```
@@ -0,0 +1,84 @@
+		return new Promise(async (resolve, reject) => {
+			try {
+				let queryObject = filterData != 'all' ? filterData : {}
+				let updatedData = await database.models.organizationExtension
+					.findOneAndUpdate(queryObject, updateData, returnData)
+					.lean()
+
+				return resolve(updatedData)
+			} catch (error) {
+				return reject(error)
+			}
+		})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)